code/unclassified/src/leap_year/leap_year.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
Please format this code according to `gofmt`, as seen [here](https://play.golang.org/p/ud8YUyAikXP).